### PR TITLE
Adds rule to allow glusterd to access RDMA socket

### PR DESCRIPTION
**Pull request by lveyde**
Resolves: rhbz#1779052

Signed-off-by: Lev Veyde <lveyde@redhat.com>

### CODE REVIEW

**Comment by lveyde** [comment]
@wrabcak @rhatdan

File: policy/modules/contrib/glusterd.te [hunk 1]
```diff
@@ -79,6 +79,7 @@ allow glusterd_t self:tcp_socket { accept listen };
 allow glusterd_t self:unix_stream_socket { accept listen connectto };
 allow glusterd_t self:rawip_socket create_socket_perms;
 allow glusterd_t self:unix_stream_socket create_stream_socket_perms;
+allow glusterd_t self:netlink_rdma_socket { bind create setopt };
```
Comment (zpytela) [inline policy/modules/contrib/glusterd.te]: Please use one of the macros available for socket permissions in https://github.com/fedora-selinux/selinux-policy/blob/rawhide/policy/support/obj_perm_sets.spt

**Review by zpytela — CHANGES_REQUESTED** [review]
@lveyde, do you have any test using this socket type?

**Comment by zpytela** [comment]
@lveyde please also ensure the same commit goes to the glusterfs-selinux package as the gluster team manages their own policy. I actually expect all future development happens only there.

**Comment by wrabcak** [comment]
@zpytela, Can you provide the link for the gluster-selinux repo, so all parties are on same page where the commit should go?

**Comment by zpytela** [comment]
https://github.com/gluster/glusterfs-selinux/

**Review by WOnder93 — APPROVED** [review]

**Review by zpytela — APPROVED** [review]

**Comment by zpytela** [comment]
Thank you for the PR; consider cherry-picking to https://github.com/gluster/glusterfs-selinux/

**Pull request merged**